hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/ScheduleServiceImpl.java

```diff
@@ -32,4 +48,73 @@ public void registerSchemas(@NonNull final SchemaRegistry registry) {
         registry.register(new V0490ScheduleSchema());
         registry.register(new V0570ScheduleSchema());
     }
+
+    @Override
+    public Iterator<ExecutableTxn> iterTxnsForInterval(
+            Instant start,
+            Instant end,
+            Supplier<StoreFactory> cleanupStoreFactory,
+            Function<TransactionBody, Set<Key>> requiredKeysFn,
+            HandleContext context) {
+        var store = cleanupStoreFactory.get().writableStore(WritableScheduleStoreImpl.class);
```

Comment:
Here imo we should use the `StoreFactory` from the `HandleContext`.

Reply:
No longer relevant as we don't pass the HandleContext now